src/Modules/OrchardCore.Commerce/Services/ShoppingCartPersistenceBase.cs

```diff
@@ -68,6 +78,12 @@ public async Task StoreAsync(ShoppingCart items)
     /// </returns>
     protected abstract Task<bool> StoreInnerAsync(string key, ShoppingCart items);
 
+    /// <summary>
+    /// Remove the items using the <see cref="ShoppingCartPersistenceBase"/>-specific <paramref name="key"/>.
+    /// </summary>
+    /// <param name="key">A prefix and (if set) the shopping cart ID combined.</param>
+    /// <returns>The removed <see cref="ShoppingCart"/> instance.</returns>
+    protected abstract Task<bool> RemoveInnerAsync(string key);
```

Comment:
The doc says it returns the removed `ShoppingCart` instance, but actually it return `bool`.
Please update the `<returns>` text.